src/supabase_client.py

```
@@ -1,11 +1,16 @@
 import os
+import logging
+from typing import Any, Dict, List, Optional
+
 import requests
 from PySide6.QtCore import QObject, Signal
```

Ruff (0.11.9)

`PySide6.QtCore.Signal` imported but unused

Remove unused import: `PySide6.QtCore.Signal`

(F401)